packages/valory/protocols/tendermint/README.md

```diff
@@ -16,10 +16,14 @@ description: A protocol for communication between two AEAs to share tendermint c
 license: Apache-2.0
 aea_version: '>=1.0.0, <2.0.0'
 speech_acts:
-  request: 
-    query: pt:optional[pt:str]  # required field, optional provides default value, not used otherwise.
-  response:
+  request_genesis_info: 
+    query: pt:optional[pt:str]
+  request_recovery_params:
+    query: pt:optional[pt:str]
+  response_genesis_info:
     info: pt:str
+  response_recovery_params:
+    params: pt:str
```

Comment:
I suggest changing to:

get_genesis_info, genesis_info

o/w there is always this bloat in the performatives

Reply:
I will address this after merging #1736.

Reply:
Done in e4555a5517fd2ea778295d7ed4105d1cde85ddd6.